src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,124 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
+import classNames from 'classnames';
 
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+import { getMovie } from '../../api/api';
+import { FindMovieShape } from './FindMovieShape';
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie = ({ addMovie, movies }) => {
+  const [foundMovie, setFoundMovie] = useState(null);
+  const [title, setTitle] = useState('');
+  const [error, setError] = useState(null);
+
+  const searchMovie = () => {
+    getMovie(title)
+      .then((movie) => {
+        if (movie.Response === 'False') {
+          setError(`Can't find a movie with such a title`);
+          setFoundMovie(null);
+
+          return;
+        }
+
+        setFoundMovie({
+          title: movie.Title,
+          description: movie.Plot,
+          imgUrl: movie.Poster,
+          imdbUrl: `https://www.imdb.com/title/${movie.imdbId}`,
+          imdbId: movie.imdbID,
+        });
+      });
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+
+    if (!movies.some(movie => movie.imdbId === foundMovie.imdbId)) {
```

Comment:
You can move this check to your addMovie function on the upper level and there will be no need to pass 'movies' here